dev-docs/Development.md

```diff
@@ -5,13 +5,29 @@
 For Golang, we try to follow [Google's code style](https://google.github.io/styleguide/go/)
 For Python we try to follow [PEP8](https://peps.python.org/pep-0008/) style guid
 
+### Hermit
+
+We use [hermit](https://cashapp.github.io/hermit/) to manage development environment.
+
+To install, [follow the official docs](https://cashapp.github.io/hermit/usage/get-started/#installing-hermit).
```

Comment:
I believe just running `source bin/activate-hermit` will do everything you need, no need to follow any instructions.

Reply:
The source bin activates the hermit in the current folder/project. The zsh integration watches where you are and activates automatically the env.

It's nice once you are working with multiple terminal sessions at the same time (my case)

But it's already mentioned as recommended in the README.hermit.md